locale/en.po

```diff
@@ -6861,11 +6873,11 @@ msgstr "No Upgrade"
 
 #: ../simulation_parameters/microbe_stage/organelles.json:34
 msgid "UPGRADE_PILUS_INJECTISOME"
-msgstr "Injectisome Pilus"
+msgstr "When in engulf mode ([thrive:input]g_toggle_engulf[/thrive:input]) this type of cilia will cause vortices in the water that pull in nearby objects. Useful for capturing prey."
```

Comment:
This text change is probably unintentional (and also a few lines below this). From what I can see this is the only issue still showing up in the file diff view.

Reply:
Huh, no idea how that happened. Should be fixed now, though.